tests/file_storage/tests.py

```diff
@@ -729,6 +729,22 @@ def test_save_overwrite_behavior_temp_file(self):
         finally:
             self.storage.delete(name)
 
+    def test_file_name_truncation(self):
+        name = "test_long_file_name.txt"
+        file = ContentFile(b"content")
+        stored_name = self.storage.save(name, file, max_length=10)
+        self.assertEqual(stored_name, "test_l.txt")
+        self.assertEqual(len(stored_name), 10)
+        self.storage.delete(stored_name)
```

Comment:
Thank you, I updated to
```python
        try:
            self.assertEqual(stored_name, "test_l.txt")
            self.assertEqual(len(stored_name), 10)
        finally:
            self.storage.delete(stored_name)
```
as this follows the structure of the other tests for this test class

> On a second look, the self.storage.delete call may not be needed since in the setUp method from OverwritingStorageTests, the following would take care of that self.addCleanup(shutil.rmtree, self.temp_dir)

Agreed, it's not strictly necessary and we could chose to refactor these tests (but as a separate PR, not for a backport 👍)